src/families/bitcoin/publicAddresses.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+/**
+ * These addresses are PUBLIC addresses
+ * We use them for tests and also for dry-run estimations
+ * DO NOT USE AS RECIPIENT OR SIGN TRANSACTION INTO
```

Comment:
```suggestion
 * DO NOT USE AS RECIPIENT OR SIGN TRANSACTIONS INTO THEM
```
Unless I'm mistaken, do you mean "sign transactions _to these accounts_" or something else?